{
  "uid" : "889ee7ae56d6f09a",
  "name" : "test_slot_length",
  "fullName" : "cardano_node_tests.tests.test_configuration#test_slot_length",
  "historyId" : "7b405f8f0831a1225012597708b90a9e",
  "time" : {
    "start" : 1603206583860,
    "stop" : 1603206583860,
    "duration" : 0
  },
  "description" : "Test the \"slotLength\" configuration.",
  "descriptionHtml" : "<p>Test the &quot;slotLength&quot; configuration.</p>\n",
  "status" : "broken",
  "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-martink/pytest-5/popen-gw4/test_configuration_py0/tmp/testrun_20201020.log'",
  "statusTrace" : "tmp_path_factory = TempPathFactory(_given_basetemp=PosixPath('/tmp/pytest-of-martink/pytest-5/popen-gw4'), _trace=<pluggy._tracing.TagTracerSub object at 0x7f3ef2c41fd0>, _basetemp=PosixPath('/tmp/pytest-of-martink/pytest-5/popen-gw4')), worker_id = 'gw4'\nrequest = <SubRequest 'cluster_manager' for <Function test_slot_length>>\n\n    @pytest.yield_fixture\n    def cluster_manager(\n        tmp_path_factory: TempdirFactory,\n        worker_id: str,\n        request: FixtureRequest,\n    ) -> Generator[parallel_run.ClusterManager, None, None]:\n>       cluster_manager_obj = parallel_run.ClusterManager(\n            tmp_path_factory=tmp_path_factory, worker_id=worker_id, request=request\n        )\n\n/home/martink/Source/repos/cardano-node-tests/cardano_node_tests/tests/conftest.py:121: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/home/martink/Source/repos/cardano-node-tests/cardano_node_tests/utils/parallel_run.py:67: in __init__\n    self.lock_log = self._init_log()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <cardano_node_tests.utils.parallel_run.ClusterManager object at 0x7f3ef2a0d070>\n\n    def _init_log(self) -> Path:\n        \"\"\"Return path to run log file.\"\"\"\n        env_log = os.environ.get(\"PARALLEL_LOG\")\n        run_log = Path(env_log or self.lock_dir / RUN_LOG_FILE).resolve()\n        # if PARALLEL_LOG env variable was set, create the log file if it doesn't exist\n        if env_log:\n>           open(run_log, \"a\").close()\nE           FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-martink/pytest-5/popen-gw4/test_configuration_py0/tmp/testrun_20201020.log'\n\n/home/martink/Source/repos/cardano-node-tests/cardano_node_tests/utils/parallel_run.py:79: FileNotFoundError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "change_dir",
    "time" : {
      "start" : 1603206583861,
      "stop" : 1603206583862,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "cluster_manager",
    "time" : {
      "start" : 1603206583864,
      "stop" : 1603206583864,
      "duration" : 0
    },
    "status" : "broken",
    "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-martink/pytest-5/popen-gw4/test_configuration_py0/tmp/testrun_20201020.log'\n",
    "statusTrace" : "  File \"/home/martink/.local/python_venvs/cardano_node/lib64/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/martink/.local/python_venvs/cardano_node/lib64/python3.8/site-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/martink/.local/python_venvs/cardano_node/lib64/python3.8/site-packages/_pytest/fixtures.py\", line 908, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/martink/Source/repos/cardano-node-tests/cardano_node_tests/tests/conftest.py\", line 121, in cluster_manager\n    cluster_manager_obj = parallel_run.ClusterManager(\n  File \"/home/martink/Source/repos/cardano-node-tests/cardano_node_tests/utils/parallel_run.py\", line 67, in __init__\n    self.lock_log = self._init_log()\n  File \"/home/martink/Source/repos/cardano-node-tests/cardano_node_tests/utils/parallel_run.py\", line 79, in _init_log\n    open(run_log, \"a\").close()\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  }, {
    "name" : "cluster_cleanup",
    "time" : {
      "start" : 1603206583862,
      "stop" : 1603206583862,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "worker_id",
    "time" : {
      "start" : 1603206583862,
      "stop" : 1603206583862,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "session_autouse",
    "time" : {
      "start" : 1603206583862,
      "stop" : 1603206583862,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "slot_length_start_cluster",
    "time" : {
      "start" : 1603206583863,
      "stop" : 1603206583864,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "tmp_path_factory",
    "time" : {
      "start" : 1603206583861,
      "stop" : 1603206583861,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "temp_dir",
    "time" : {
      "start" : 1603206583863,
      "stop" : 1603206583863,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : "Test the \"slotLength\" configuration.",
    "status" : "broken",
    "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-martink/pytest-5/popen-gw4/test_configuration_py0/tmp/testrun_20201020.log'",
    "statusTrace" : "tmp_path_factory = TempPathFactory(_given_basetemp=PosixPath('/tmp/pytest-of-martink/pytest-5/popen-gw4'), _trace=<pluggy._tracing.TagTracerSub object at 0x7f3ef2c41fd0>, _basetemp=PosixPath('/tmp/pytest-of-martink/pytest-5/popen-gw4')), worker_id = 'gw4'\nrequest = <SubRequest 'cluster_manager' for <Function test_slot_length>>\n\n    @pytest.yield_fixture\n    def cluster_manager(\n        tmp_path_factory: TempdirFactory,\n        worker_id: str,\n        request: FixtureRequest,\n    ) -> Generator[parallel_run.ClusterManager, None, None]:\n>       cluster_manager_obj = parallel_run.ClusterManager(\n            tmp_path_factory=tmp_path_factory, worker_id=worker_id, request=request\n        )\n\n/home/martink/Source/repos/cardano-node-tests/cardano_node_tests/tests/conftest.py:121: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/home/martink/Source/repos/cardano-node-tests/cardano_node_tests/utils/parallel_run.py:67: in __init__\n    self.lock_log = self._init_log()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <cardano_node_tests.utils.parallel_run.ClusterManager object at 0x7f3ef2a0d070>\n\n    def _init_log(self) -> Path:\n        \"\"\"Return path to run log file.\"\"\"\n        env_log = os.environ.get(\"PARALLEL_LOG\")\n        run_log = Path(env_log or self.lock_dir / RUN_LOG_FILE).resolve()\n        # if PARALLEL_LOG env variable was set, create the log file if it doesn't exist\n        if env_log:\n>           open(run_log, \"a\").close()\nE           FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-martink/pytest-5/popen-gw4/test_configuration_py0/tmp/testrun_20201020.log'\n\n/home/martink/Source/repos/cardano-node-tests/cardano_node_tests/utils/parallel_run.py:79: FileNotFoundError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "fda6a8073208411e",
      "name" : "log",
      "source" : "fda6a8073208411e.txt",
      "type" : "text/plain",
      "size" : 117
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "cluster_cleanup::0",
    "time" : {
      "start" : 1603206584157,
      "stop" : 1603206584158,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "temp_dir::0",
    "time" : {
      "start" : 1603206583892,
      "stop" : 1603206583892,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "tag",
    "value" : "@pytest.mark.usefixtures('temp_dir')"
  }, {
    "name" : "parentSuite",
    "value" : "cardano_node_tests.tests"
  }, {
    "name" : "suite",
    "value" : "test_configuration"
  }, {
    "name" : "host",
    "value" : "bender-3900x"
  }, {
    "name" : "thread",
    "value" : "273064-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "cardano_node_tests.tests.test_configuration"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ {
    "name" : "https://github.com/input-output-hk/cardano-node-tests/blob/26dda6fa7b63cb50e400fc7bdd3b38a9e569a112/cardano_node_tests/tests/test_configuration.py#L120",
    "url" : "https://github.com/input-output-hk/cardano-node-tests/blob/26dda6fa7b63cb50e400fc7bdd3b38a9e569a112/cardano_node_tests/tests/test_configuration.py#L120",
    "type" : "link"
  } ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ "@pytest.mark.usefixtures('temp_dir')" ]
  },
  "source" : "889ee7ae56d6f09a.json",
  "parameterValues" : [ ]
}